Update README.md (#14)
vtex-apps/order-payment: docs/README.md

> 📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/order-payment) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> # ORDER PAYMENT

Reviewer: ```suggestion
# Order Payment
```

> ⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️

Reviewer: ```suggestion
> :warning: *Although ready to be installed, the Order Payment app is currently an **open-beta project**. Bear in mind that due to that status, you can expect an accelerated state of evolution.*
```

<!-- DOCS-IGNORE:start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- DOCS-IGNORE:end -->

> TBC

Reviewer: Can't we add some description to clarify the app functionality?

> TBC

Reviewer: Can we add some description to clarify the app functionality?

Author: Not for now 😔  it is under construction yet, many things might change, it is not even on beta.

## Configuration 

> TBC

Reviewer: ```suggestion
🚧 To be defined.
```

## Customization

> TBC

Reviewer: ```suggestion
🚧 To be defined.
```
